Core/include/Acts/EventData/TrackStatePropMask.hpp

```diff
@@ -31,8 +31,7 @@ enum struct TrackStatePropMask : uint8_t {
   Smoothed = 1 << 2,
   Jacobian = 1 << 3,
 
-  Uncalibrated = 1 << 4,
-  Calibrated = 1 << 5,
+  Calibrated = 1 << 4,
```

Comment:
So this enum predates the refactoring done in this PR. Previously, there was the option to not allocate storage for an uncalibrated measurement. In the meantime, uncalibrated measurement switched to mean `const SourceLink&`, which is a 64bit pointer. In this PR, I'm removing this possibility because I'm touching this code anyway.

A track state can have all of these components at the same time, and it will always have the remaining components.

Reply:
Can we keep this as is for now @tboldagh?

Reply:
Sure.